ehsm_kms_service/function.js

```diff
@@ -37,13 +37,17 @@ const base64_decode = (base64_str) =>
 
 /**
  * Clear nonce cache for more than <NONCE_CACHE_TIME> minutes
- * @returns nonce_cache_timer
+ * nonce_database[appid]
+ *  - type: array
+ *  - sort: [new timestamp, old timestamp, ...]
+ * @returns nonce_cache_timer, nonce_database
  */
 const _nonce_cache_timer = () => {
   const nonce_database = {}
   const timer = setInterval(() => {
     try {
       for (const appid in nonce_database) {
+        // slice_index  Index of the cache that exceeded the maximum time
         let slice_index =
           nonce_database[appid] &&
           nonce_database[appid].findIndex((nonce_data) => {

```

Comment:
Will findIndex() return one or all matched results?

Reply:
Returns the position of the first element of the qualified array